test/visibility/import/edgeCases/importChain2.future

```diff
@@ -0,0 +1 @@
+Bug: Can't import a list of reexported symbols
```

Comment:
We should open an issue to implement that, that's something I can see folks potentially wanting to do.

Reply:
Also, good catch :)

Reply:
https://github.com/Cray/chapel-private/issues/886